src/main/kotlin/no/nav/dagpenger/regel/api/db/PostgresBruktSubsumsjonStore.kt

```diff
@@ -37,6 +43,8 @@ class PostgresBruktSubsumsjonStore(private val dataSource: HikariDataSource) : B
                         "Vedtak", subsumsjonBrukt.arenaTs
                     ).asUpdate
                 )
+                insertCounter.inc()
```

Comment:
Kan chaine med also for å slippe scope leak av variabel

Reply:
Takk, det stemmer :)